doc/index.rst

```diff
@@ -1,41 +1,112 @@
 
-.. .. _lsst.ts.dsm-using:
+  Sections
+  ========
 
-.. Using lsst.ts.dsm
-.. =================
+  and
 
-.. toctree linking to topics related to using the module's APIs.
+  Subsections
+  -----------
 
-.. .. toctree::
-..    :maxdepth: 1
+  and
 
-.. _lsst.ts.dsm-contributing:
+  Subsubsections
+  ^^^^^^^^^^^^^^
 
-Contributing
-============
+  To add images, add the image file (png, svg or jpeg preferred) to the
+  images/ directory. The reST syntax for adding the image is
 
-``lsst.ts.dsm`` is developed at https://github.com/lsst-ts/ts_dsm.
-You can find Jira issues for this module under the `ts_dsm <https://jira.lsstcorp.org/issues/?jql=project%20%3D%20DM%20AND%20component%20%3D%20ts_dsm>`_ component.
+  .. figure:: /images/filename.ext
+   :name: fig-label
 
-.. If there are topics related to developing this module (rather than using it), link to this from a toctree placed here.
+  Caption text.
 
-.. .. toctree::
-..    :maxdepth: 1
+  Feel free to delete this instructional comment.
 
-.. _lsst.ts.dsm-pyapi:
+.. Fill out data so contacts section below is auto-populated
+.. |CSC_developer| replace::  *Replace-with-name-and-email*
+.. |CSC_product_owner| replace:: *Replace-with-name-and-email*
 
-Python API reference
-====================
+.. Note that the "ts_" prefix is omitted from the title
 
-.. automodapi:: lsst.ts.dsm
-   :no-main-docstr:
-   :no-inheritance-diagram:
+#########################
+DSM
+#########################
+
+.. update the following links to point to your CSC
+.. image:: https://img.shields.io/badge/SAL-API-gray.svg
+    :target: https://ts-xml.lsst.io/sal_interfaces/DSM.html
+.. image:: https://img.shields.io/badge/GitHub-gray.svg
+    :target: https://github.com/lsst-ts/ts_dsm
+.. image:: https://img.shields.io/badge/Jira-gray.svg
+    :target: https://jira.lsstcorp.org/issues/?jql=labels+%3D+ts_dsm
+.. image:: https://img.shields.io/badge/Jenkins-gray.svg
+    :target: https://tssw-ci.lsst.org/job/LSST_Telescope-and-Site/job/ts_dsm/
+
+.. _Overview:
+
+Overview
+========
+
+The Dome Seeing Monitor (DSM) CSC is responsible for taking output telemetry
+files from the Dome Seeing Monitor UI (need doc link) and sending the information via SAL
+for recording in the EFD. Although the DSM is part of the Environmental Awareness
+System, it currently runs stand-alone and is not tied in to the EAS in any manner.
+
+As with all CSCs, information on the package, developers and product owners can be found in the `Master CSC Table <ts_xml:index:master-csc-table:DSM>`_.
+
+.. note:: If you are interested in viewing other branches of this repository append a `/v` to the end of the url link. For example `https://ts_dsm.lsst.io/v/`
+
+
+.. _User_Documentation:
+
+User Documentation
+==================
+
+User-level documentation, found at the link below, is aimed at personnel looking to perform the standard use-cases/operations with the DSM.
+
+.. toctree::
+    user-guide/user-guide
+    :maxdepth: 2
```

Comment:
Your User Guide is reasonably short; please consider making it part of this page instead of a separate page. That makes the most used information most readily available. This is especially useful because the links in the navigation bar along the left edge can't jump to other pages (which is really frustrating).

See https://ts-atdome.lsst.io for an example.

Reply:
Note that this is explicitly allowed in the doc design. The template assumes a separate page, but that is no barrier to doing it the other way.